superbench/benchmarks/micro_benchmarks/gpu_burn_test.py

```diff
@@ -0,0 +1,143 @@
+            return False
+        
+        if not self._set_binary_path():
+            return False
+        
+        command = os.path.join(self._args.bin_dir, self._bin_name)
+        
+        if self._args.doubles:
+            command+=' -d'
+        
+        if self._args.tensor_core:
+            command+=' -tc'
+        command+= ' {} '.format(self._args.time)
+        #copy compare.ptx which needs to be in the working directory
+        compare_copy="cp " + self._args.bin_dir+ "/compare.ptx ./"
+        #remove compare.ptx from working directory
+        compare_rm="rm " + "compare.ptx"
+
+        self._commands.append(compare_copy + " && " + command + " && " + compare_rm)
+
+        return True
+
+    def _process_raw_result(self, cmd_idx, raw_output):    
+        """Function to parse raw results and save the summarized results.
+          self._result.add_raw_data() and self._result.add_result() need to be called to save the results.
+        Args:
+            cmd_idx (int): the index of command corresponding with the raw_output.
+            raw_output (str): raw output string of the micro-benchmark.
+        Return:
+            True if the raw output string is valid and result can be extracted.
+        """
+
+        #self._result.add_raw_data('raw_output_gpuburn', raw_output)
+        content = raw_output.splitlines()
+        gpu_res=[]
+        abort=False
+        failure_msg="unknown failure"
+        index=-1
+        try:
+            for idx, line in enumerate(content):
+                if 'No clients are alive!' in line or 'Couldn\'t init a GPU' in line or 'Failure during compute' in line or 'Low mem for result' in line:
+                    abort=True
+                    failure_msg=line
+                    break
+                if 'done' in line:
+                    index=idx
+                    break
+
+            if not abort:
+                if 'done' not in content[index]:
+                    abort=True
+                    failure_msg= "The result format invalid"
+                    raise failure_msg
+
+                content = content[index + 2:len(content):]
+                
+                for line in content:
+                    if 'Tested' in line:
+                        continue;
+                    if 'GPU' in line:
+                        gpu_res.append(line.strip('\n').strip('\t'))
+
+                self._result.add_result('GPU_Burn_Time',self._args.time)
+                for res in gpu_res:
+                    if 'OK' in res:
+                        self._result.add_result(res.split(':')[0].replace(' ','_') + '_Pass', 1 )
+                    else:
+                        self._result.add_result(res.split(':')[0].replace(' ','_') + '_Fail', 1 )
+                    self._result.add_raw_data('GPU-Burn_result',res)
+            else:
+                self._result.add_raw_data('GPU Burn Failure: ', failure_msg)
+                self._result.add_result('GPU-Burn_Fail', 1 )
```

Comment:
can we use lower case for metrics? besides,"gpu-burn/" will be added in front of all metrics so add_result('fail', 1 ) should be fine.

Reply:
Okay. I'll make the change.

Reply:
change made